src/css/controls/imports/nextup.less

```diff
@@ -13,11 +13,46 @@
     visibility: hidden;
     width: 100%;
 
-    .jw-flag-small-player &,
     .jw-settings-open &,
     .jw-info-open & {
         display: none;
     }
+
+    .jw-flag-small-player & {
+        padding: 0;
```

Comment:
I know this is done to spec in the design mocks, but it's jarring. Look at the right padding at breakpoint 2+. The right vertically aligns with the time slider, not the edge of the player.
<img width="416" alt="Screen Shot 2019-04-04 at 3 34 55 PM" src="https://user-images.githubusercontent.com/4603952/55584472-43bc5080-56f2-11e9-8388-5d6aae4c78d0.png">
<img width="439" alt="Screen Shot 2019-04-04 at 3 35 06 PM" src="https://user-images.githubusercontent.com/4603952/55584544-69495a00-56f2-11e9-8131-dd51352af644.png">
`padding: 0 12px 0 0;` would do the trick.